src/dxvk/imgui/dxvk_imgui.cpp

```diff
@@ -1238,39 +1301,46 @@ namespace dxvk {
       ImGui::SetCursorPosX(x + startX + (thumbnailSize - extent.x) / 2.f);
       ImGui::SetCursorPosY(y + (thumbnailSize - extent.y) / 2.f);
 
-      if (ImGui::ImageButton(pair.second.texID, extent)) {
-        const char* action;
-        if (data.find(pair.first) != data.end()) {
-          data.erase(pair.first);
-          action = "removed";
-        }
-        else {
-          data.insert(pair.first);
-          action = "added";
-        }
-
-        char buffer[256];
-        sprintf_s(buffer, "%s - %s %016llX\n", uniqueId, action, pair.first);
-        Logger::info(buffer);
+      if (ImGui::ImageButton(pair.second.texID, extent) && isListFiltered) {
+        toggleTextureSelection(pair.first, uniqueId, listRtxOption.textureSetOption->getValue());
       }
 
       if (ImGui::IsItemHovered()) {
         std::stringstream formatName;
         formatName << imageInfo.format;
 
+        //list all selections for this texture
+        std::string rtxTextureSelection;
```

Comment:
Need to handle the case where texture is unassigned, and show that to the user.  e.g. 

`Current categories:
 - None`